metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/dsl/functions/resolver/BaseFunctionResolver.java

```diff
@@ -242,4 +242,18 @@ public static StellarFunction createFunction(Class<? extends StellarFunction> cl
       return null;
     }
   }
+
+  @Override
+  public BaseFunctionResolver withInstance(StellarFunction function) {
+    // perform function resolution on the instance that was passed in
```

Comment:
Allows us to inject a `StellarFunction` instance into the set of resolved functions.  

This is what solves the problem of how do we ensure that Stellar functions and code running in a Storm topology "see" the same underlying HBase data, without being able to run HBase and no longer having the mock HBase logic that existed previously.